experiences/src/main/kotlin/io/rover/campaigns/experiences/EventReceiver.kt

```diff
@@ -17,13 +19,19 @@ import io.rover.sdk.services.EventEmitter
  */
 open class EventReceiver(
     private val eventEmitter: EventEmitter?,
-    private val eventQueueService: EventQueueServiceInterface
+    private val eventQueueService: EventQueueServiceInterface,
+    private val userInfo: UserInfoInterface
 ) {
     open fun startListening() {
         eventEmitter?.let {
             eventEmitter.trackedEvents.subscribe { event ->
                 eventQueueService.trackEvent(transformEvent(event), "rover")
             }
+            eventEmitter.trackedEvents.subscribe { event ->
```

Comment:
May as well just subscribe once and do `getConversion` in same  `onNext` block as `trackEvent`